optimum/intel/openvino/training_args.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass, field
+
+from transformers import TrainingArguments
+
+
+@dataclass
+class OVTrainingArguments(TrainingArguments):
+    """
+    Arguments pertaining to OpenVINO/NNCF-enabled training flow
+    """
+
+    nncf_compression_config: str = field(
```

Comment:
This is related to the previous two comments. Since distillation is always use-case and topology specific it makes sense not to introduce it in the common OVTrainingArguments but rather create a specific Trainer object for the method, i.e. `JPQDQuestionAnsweringTrainer` and have all the distillation-specific parameters there only, it could be additional `OptimizationArguments` as proposed by @echarlaix. 
It is also possible to have a base `JPQDTrainer` and inherit from it every time.
@helena-intel, @echarlaix, @vuiseng9, what do you think?

Reply:
I like a `JPQDTrainer` and/or `JPQDQuestionAnsweringTrainer`. For "regular" QA we already have a specific trainer too, because question answering requires some specific data processing, so to me a `JPQDTrainer` would be in line with that.

I do really like the option to add a `nncf_compression_config` in the trainer command line though, and I would like to consider adding that to the OVTrainer as an optional field (and also adapt the existing examples). The default config is designed to work well most of the time, but there can always be cases where you want to experiment with the config, or want to make other tradeoffs for speed/accuracy/model size and users should have that option.

Reply:
Not sure to understand why we would need a `JPQDTrainer`, in my opinion it would be easier from a user perspective to have one unique `OVTrainer` which can be used to combine any of the compression techniques it enables

Reply:
In my opinion we should have one class (`OVTrainer`), allowing users to either apply quantization, pruning, knowledge distillation or any combination of these techniques. I don't see the gain of having a specific class for JPQD but I might be missing an important point

Reply:
@echarlaix, I see your point. But, in my opinion, this is not a practical approach to let the user try various combinations of optimization methods while we know the best one. For example, using pruning without quantization does not make sense because the latter has certain advantages, namely, you can quickly quantize the model, it does not hurt accuracy a lot, and you can get a decent speedup after quantization, it does not require training HW in the case of PTQ. So that the workflow here should be quantized first and then use pruning on top of quantization if you still are not satisfied with the performance.
Let's discuss this problem.